### PR TITLE
add use of real pcd files and related instructions.

### PR DESCRIPTION
**Pull request by AustinDeric**
Also make resolution of fictional point clouds to 0.5mm

### CODE REVIEW

File: godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch [hunk 1]
```diff
@@ -9,6 +9,8 @@
   <arg name="robot_model_plugin" default="abb_irb2400_descartes/AbbIrb2400RobotModel"/>
   <arg name="debug_rviz" default="false"/>
   <arg name="debug_core" default="false"/> <!--Brings up the surface blending service in debug mode-->
+  <arg name="real_pcd" default="false"/>
+  <arg name="pcd_location" default="$(env HOME)/godel_point_cloud_data/surface4.pcd"/> <!-- path to pcd file of part -->
```
Comment (Jmeyer1292) [inline godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch]: I would remove the default directory unless it points to something in the Godel repository

File: godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch [hunk 2]
```diff
@@ -47,10 +49,16 @@
 
   <!-- If the user specifies a 'fake' sensor, publish fake data clouds -->
   <group if="$(arg sim_sensor)">
-    <remap from="generated_cloud" to="sensor_point_cloud"/>
-    <node pkg="godel_surface_detection" type="point_cloud_generator_node" name="point_cloud_generator_node">
-      <rosparam command="load" file="$(find godel_irb2400_support)/config/point_cloud_descriptions.yaml"/>
-    </node>
+    <group if="$(arg real_pcd)">
+      <remap from="cloud_pcd" to="sensor_point_cloud"/>
+      <node pkg="pcl_ros" type="pcd_to_pointcloud" name="point_cloud_generator_node" args="$(arg pcd_location) 0.5" />
```
Comment (Jmeyer1292) [inline godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch]: Point clouds will come in different reference frames, so let's set the `frame_id` parameter of this node.

File: godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch (hunk 1 above)
Comment (Jmeyer1292) [reply godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch]: And make the argument pcd location have an `if=$(arg real_cd)` clause.

Comment (Jmeyer1292) [reply godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch]: This way the user gets a nice error message whenever they forget to specify a filepath

Comment (AustinDeric) [reply godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch]: All fixed and tested: irb2400_blending.launch] requires the 'pcd_location' arg to be set

**Pull request merged**